ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -1543,11 +1586,17 @@ static void ggml_vk_load_shaders(vk_device& device) {
         device->pipeline_matmul_id_f32 = std::make_shared<vk_matmul_pipeline_struct>();
     }
 
+    vk::PhysicalDeviceProperties2 props2;
+    device->physical_device.getProperties2(&props2);
+    std::string device_name = props2.properties.deviceName.data();
```

Comment:
I don't think this is needed anymore (also, the device name is available in `device->name`, and the properties in `device->properties`)

Reply:
I forgot to remove that from some previous tests, thanks for pointing it out.

Reply:
If I don't set the device name like this the shader executes at wave64 speed for some reason.

Reply:
I checked and `device->name` contains just something like `Vulkan0`, so that's why. But if you need the actual device name, it should be stored in the device struct on init and only accessed here, something like `physica_device_name`.

But as mentioned in my other comment, we can probably ignore device names for now.